schema.sql

```diff
@@ -11,4 +11,24 @@ CREATE TABLE animals (
 
 Alter TABLE animals
   ADD COLUMN species varchar(100);
-  
\ No newline at end of file
+--------------------------------------------------------
+-- create table owners--
+CREATE TABLE owners (
+    id integer PRIMARY KEY,
```

Comment:
Optional  - its best practice to add `IDENTITY(1,1)` to your database to increment items in your database to easily keep track of items in your db 😄

Reply:
Thanks for the heads up.